content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -315,6 +315,12 @@ tolerations to all daemons, to prevent DaemonSets from breaking.
 Adding these tolerations ensures backward compatibility. You can also add
 arbitrary tolerations to DaemonSets.
 
+{{< note >}}
+If you manually schedule a pod by defining a node name in your workload yaml spec
+file like a pod, deployments, etc. it will override the usual scheduling mechanism,
+hence, bypassing the taint.
```

Comment:
There are a few nits in this note. For example, we don't say "workload yaml spec file",
we say "workload manifest". "... like a pod, deployments" are inaccurate. When mentioning a resource kind, use the resource names is preferred, i.e. "Pod" and "Deployment". The scheduling mechanism is not overridden. I don't think this claim is correct. If the `nodeName` for a Pod manifest is manually specified, the scheduler will assign the Pod to the node specified, regardless of any taints or tolerations. I think this is what you were trying to clarify.

Reply:
Nice, thanks for the review @tengqm. Yes, that was what I was trying to clarify. What do you think about the correction now?

>{{< note >}}
If you manually schedule a pod by defining a `nodeName` in your workload manifest like a "Pod" and "Deployment", the scheduler will assign the Pod to the node specified, regardless of any taints or tolerations.
{{< /note >}}